src/components/data_search/DatasetFilterList.tsx

```diff
@@ -64,6 +64,8 @@ interface FilterItemRangeProps {
   max?: number
   minCategory: string
   maxCategory: string
+  minInputProps?: React.InputHTMLAttributes<HTMLInputElement>
+  maxInputProps?: React.InputHTMLAttributes<HTMLInputElement>
```

Comment:
It doesn't look like we're using these properties:

<img width="690" height="381" alt="Image" src="https://github.com/user-attachments/assets/c964e321-227d-4fd5-87a3-9163404c8425" />